### ISSUE TITLE
Handle large XML files

### ISSUE THREAD

**Issue by yvrng** [kestra-io/plugin-serdes#66]
### Feature description

I would like to use the `io.kestra.plugin.serdes.xml.XmlReader` to read large XML files, without increasing the memory R/L of the workers

**Comment by anna-geller** [comment]
quite an old issue, is this still relevant for you @yvrng?

**Comment by yvrng** [comment]
@anna-geller, yes, it's still relevant, we have limitations on processing large XML/JSON files. In the meantime, we use a worker group with a large amount of RAM, but it's not ideal.

However, we're not stuck for now. I'll come back to you when the need arises again, for a possible co-dev: we have a few ideas of our own :)

**Comment by fdelbrayelle** [comment]
Hi @yvrng 👋 This was fixed and QAed with [plugin-serdes v1.5.0](https://github.com/kestra-io/plugin-serdes/releases/tag/v1.5.0) 🎉
